src/app/styles/constants/buttonCoreStyles.js

```diff
@@ -0,0 +1,64 @@
+import { stylesCore } from './stylesCore';
+
+export const buttonColorVariant = {
```

Comment:
🟠 aren't `Button` related styles are suppose to be near `Button` component?

Reply:
Thanks for comment! Done.